RELEASE_PROCESS.md

```diff
@@ -49,30 +49,37 @@ Starting download https://packages.chef.io/files/current/chef-server/14.2.23/ubu
 
 ### Testing the Release
 
-Every merge to chef-server master must be built, and this build must be tested with the full Umbrella automated integration test pipeline at https://buildkite.com/chef/chef-umbrella-master-chef-server-full.
+Every merge to chef-server master must be built, and this build must be tested with the full Umbrella automated integration test pipeline at https://buildkite.com/chef/chef-umbrella-master-chef-server-full (NOTE: use https://buildkite.com/chef/chef-umbrella-master-chef-server for testing the 12.17.15 upgrade path).
 The integration test run for the tag being shipped must be successful.
 
 Any Chef Infra Server release 12.17.15 or later should be able to upgrade directly to the latest release of 14. The nightly builds test upgrades to the latest current artifact from 12.17.15 and 13.2.0.  Releases prior to 12.17.15 must perform a stepped upgrade.  See: https://docs.chef.io/server/upgrades/#upgrade-matrix
 
-Test that the current stable release can successfully upgrade to the new release.
-
 Past stable releases:
 https://downloads.chef.io
 
+Using the step-by-step Umbrella testing process detailed below...
+
+1. Test that the 12.17.15 release can successfully upgrade to the new release (for this, use the pipeline at https://buildkite.com/chef/chef-umbrella-master-chef-server).
+1. Test that the 13.2.0   release can successfully upgrade to the new release.
+1. Test that the latest stable release can successfully upgrade to the new release.
+
 Umbrella Testing Step-by-Step:
 
-1. Navigate your web browser to https://buildkite.com/chef/chef-umbrella-master-chef-server
+1. Navigate your web browser to https://buildkite.com/chef/chef-umbrella-master-chef-server-full
 1. Select 'New Build'.
 1. Leave 'Branch' set to 'master'.
 1. Select 'Options' to expand the 'Environment Variables' field.
-1. Enter `INSTALL_VERSION=<version number>` into the 'Options | Environment Variables' field, where _version number_ is the stable  version number of the release you wish to test upgrading FROM (for example 14.0.65).
-1. Enter `UPGRADE_VERSION=<version number>` into the 'Options | Environment Variables' field, where _version number_ is the current version number of the release candidate you wish to test upgrading TO (for example 14.2.2).
+1. Enter `INSTALL_VERSION=<version number>` into the 'Options | Environment Variables' field, where _version number_ is the stable channel  version number of the release you wish to test upgrading FROM (for example 14.0.65).
```

Comment:
might want to update the numbering on these?

Reply:
I updated it to reflect the 12.17.15 -> YOUR-RELEASE path.